Classes/Dto/FocusKeywordModuleConfigurationDto.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace NEOSidekick\AiAssistant\Dto;
+
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * @Flow\ValueObject
+ * @Flow\Proxy(false)
+ */
+final class FocusKeywordModuleConfigurationDto
+{
+    private const DEFAULT_LIMIT = 5;
+    /**
+     * @var string
+     */
+    protected string $workspace;
+
+    /**
+     * 'only-empty', 'only-existing' or 'both'
+     * @var string
+     */
+    protected string $mode;
+
+    /**
+     * @var bool
```

Comment:
Is there any reason to use doc comments, when the type is already defined in PHP? (really dont know=

Reply:
Ich glaube bei primitiven Typen wäre das echt egal. Bei Objekten hatten wir da aber Kompabilitätsprobleme mit Neos 7, deshalb versuche ich das generell überall im Docblock anzugeben. Doppelt gemoppelt, aber egal.

Reply:
Oh, was neues zu lernen. Wie verwendet Neos diese Kommentare?